libr/anal/cc.c

```diff
@@ -15,6 +15,39 @@ R_API void r_anal_cc_del(RAnal *anal, const char *name) {
 	}
 }
 
+static const char *getreg(RAnal *anal, const char *alias) {
```

Comment:
sounds like a good `r_reg_get_name_by_type`

Reply:
You want me to move this into RReg?

Reply:
i will take your silence as a yes

Reply:
sorry, I missed the notification I guess :D but yeah